### PR TITLE
DROOLS-5349: Add tooltip for GDST attribute Default value

### PR DESCRIPTION
**Pull request by jomarko**
It is not intuitive that Default value change will affect just new rows of table. This PR add a explanation tooltip.

For more details see: https://issues.redhat.com/browse/DROOLS-5349

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties [hunk 1]
```diff
@@ -36,6 +36,7 @@ ValueList=Value list
 ValueListsExplanation=Value lists are an optional comma separated list of values to show as a drop down.
 optionalValueList=(optional) value list:
 DefaultValue=Default value
+DefaultValueExplanation=Change of ''Default value'' will have effect just on new table rows.
```
Comment (dupliaka) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties:39]: Proposal: to change this title on  "Change of ''Default" value affects only the new table rows."

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties (hunk 1 above)
Comment (jomarko) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties]: Thank you, applied

**Comment by dupliaka** [comment]
Could you please check why does it not disappears if I switch between Model and Columns tabs:
image

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
@dupliaka thank you for spotting, great catch. 

Unfortunately in my fix I use just built int html tooltips. Functionality is same but no so cool design as in screenshots. The issue with 'PopoverUtils' I used previously is thee need to call `destroyPopovers()` mehthod in the right moment. 

`PopoverUtils` are used mostly in 'New Decision Table Wizard' where is the needed moment for calling `destroyPopovers()` quite straightforward. it  is in moment of closing wizard.

However for this columns preview components I was not able to find proper place for estroying popovers. As guided decision tables editor is not the top priority currently, I decided to use just built in tooltip.

**Comment by jomarko** [comment]
jenkins please retest this

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties [hunk 2]
```diff
@@ -36,6 +36,7 @@ ValueList=Value list
 ValueListsExplanation=Value lists are an optional comma separated list of values to show as a drop down.
 optionalValueList=(optional) value list:
 DefaultValue=Default value
+DefaultValueExplanation=Change of ''Default value'' will affect just the new table rows.
```
Comment (dupliaka) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties:39]: Are you sure that there should be a period? I see some notifications are with it and without that makes it strange.
And the neighbor notification got no period as well WDYT?
Period

Comment (jomarko) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties]: @dupliaka thank you, applied in a separate commit.

**Comment by jomarko** [comment]
@kiegroup/gatekeepers please merge.

**Pull request merged**